core/settings/qPref.cpp

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qPref_private.h"
+#include "qPref.h"
+
+
+qPref *qPref::m_instance = NULL;
+qPref *qPref::instance()
+{
+	if (!m_instance)
```

Comment:
I think we discussed this already, but I forgot: Any reason not to use the standard singleton idiom:
```
qPref *qPref::instance()
{
	qPref self;
	return &self;
}
```
If this doesn't work for some subtle reason then it needs a comment.